android/apps/AutoTesterAndroid/app/src/main/java/com/mousebirdconsulting/autotester/TestCases/ClusteredMarkersTestCase.kt

```diff
@@ -30,19 +30,23 @@ class ClusteredMarkersTestCase(activity: Activity?) :
 
     @Throws(Exception::class)
     override fun setUpWithMap(mapVC: MapController): Boolean {
+        baseCase.onVectorsLoaded = { vectors ->
```

Comment:
Vectors test adds asynchronously now, and this test was using the collection before it was populated.